client/src/selectors/stake.js

```diff
@@ -34,5 +34,15 @@ export const getPendingStakes =
 			return [];
 		}
 
-		return stakes.delegations.filter((stake) => stake.status === 'pending');
+		return stakes.delegations.filter((stake) => stake.status === 'pending' || stake.status === 'undelegating');
```

Comment:
I saw status values can be put into enum type and re-use everywhere. This avoids human typing error.
e.g:
```
// some common constants js file
export const STATUS = {
  PENDING: 'pending',
  UNDELEGATING: 'undelegating'
}
```